### DISCUSSION TITLE
How to skip Http Server Starting for a Batch/Event Consumer like app

### DISCUSSION THREAD

**Discussion by subash89** [micronaut-projects/micronaut-core#10976] [Q&A] [answered]
Hi,
I am trying to use micronaut as a SQS consumer. For this I don't need Netty/SSL server listening on a port.
But only need the dependency injection(DI) and http client to be working.

**Comment by yawkat** [comment, marked as answer]
Just don't include the micronaut-http-server-netty dependency.
